src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -344,6 +344,7 @@ def load_arguments(self, _):
         c.argument('patch_mode', arg_type=get_enum_type(self.get_models('InGuestPatchMode')), min_api='2020-06-01',
                    help='Mode of in-guest patching to IaaS virtual machine. Possible values are: Manual - You  control the application of patches to a virtual machine. You do this by applying patches manually inside the VM. In this mode, automatic updates are disabled; the paramater --enable-auto-update must be false. AutomaticByOS - The virtual machine will automatically be updated by the OS. The parameter --enable-auto-update must be true. AutomaticByPlatform - the virtual machine will automatically updated by the OS. The parameter --enable-agent and --enable-auto-update must be true')
         c.argument('ssh_key_name', help='Use it as public key in virtual machine. It should be an existing SSH key resource in Azure.')
+        c.argument('enable_hotpatching', arg_type=get_three_state_flag(), help='Patch VMs without requiring a reboot. --enable-agent must be set and --patch-mode must be set to AutomaticByPlatform', min_api='2020-12-01')
```

Comment:
didn't find the check that `enable-hostpatching` need to work with `enable-agent` and `patch-mode`, could you pls help to add it?

Reply:
Service always does it. It responses faster if added in client side. I'll check --enable-agent must be set. It even requires a special image.

Reply:
yes, let's do it later. not necessary in this bug PR. :)

Reply:
OK